src/NewTwitchApi/NewTwitchApi.php

```diff
@@ -50,29 +50,31 @@ class NewTwitchApi
     private $videosApi;
     private $webhooksApi;
     private $webhooksSubscriptionApi;
+    private $requestGenerator;
 
     public function __construct(Client $helixGuzzleClient, string $clientId, string $clientSecret, Client $authGuzzleClient = null)
     {
+        $this->requestGenerator = new RequestGenerator();
```

Comment:
Does this need to be a class field, or could it just be a local variable in the function?

Reply:
Just to confirm, you'd prefer it to be `$requestGenerator = new RequestGenerator();`? If so, yes, this works. Please confirm and I'll merge.